src/api/handlers.ts

```diff
@@ -15,14 +15,15 @@
 // along with procaptcha.  If not, see <http://www.gnu.org/licenses/>.
 import {AxiosResponse} from "axios";
 
+
 export class ProsopoApiError extends Error {
     constructor(error: AxiosResponse, context?: string, ...params: any[]) {
-
+        console.log(error)
```

Comment:
is this needed? we log the error below

Reply:
No its not, this was debug because this error handler is actually broken if there is no `.message` part. This happens frequently. I'll create follow up task to fix it.